cytominer_database/write.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
there are a TON of imports in this file. But it looks like we only use a subset of them. Lets remove the imports we aren't using

Reply:
In this scripts the import isn't even used anymore.

Reply:
Will search for all os imports

Reply:
Just importing os.listdir works, it's more complicated with os.path though (not actually a submodule of a `os` package, " The `os.path` name is an alias for this
module on Posix systems; on other systems (e.g. Mac, Windows),
os.path provides the same operations in a manner specific to that
platform, and is an alias to another module (e.g. macpath, ntpath)", see also [here](https://stackoverflow.com/questions/2724348/should-i-use-import-os-path-or-import-os)). 
Importing os.path throws an error.

Reply:
---> os [documentation](https://docs.python.org/3/library/os.path.html#:~:text=The%20os.,one%20of%20the%20different%20formats.).

Reply:
[TODO]: Exchange os.path operations with the [pathlib](https://docs.python.org/3/library/pathlib.html#module-pathlib) modules whenever possible.

Reply:
It looks like all changes in imports need to reflected in setup.py . I.e. replacing `import click` with `import click.MultiCommand` ?

Reply:
Decreasing the import `import os` --> `import os.path` worked for *some* files: `ingest.py`, tableSchema.py.
Removed imports completely when not used.